app/src/main/java/com/sopt/androidstudy/MusicPlayerService.kt

```diff
@@ -0,0 +1,104 @@
+package com.sopt.androidstudy
+
+import android.app.Notification
+import android.app.Service
+import android.content.Intent
+import android.media.MediaPlayer
+import android.os.Binder
+import android.os.Build
+import android.os.IBinder
+import android.util.Log
+import androidx.annotation.RequiresApi
+import com.sopt.androidstudy.presentation.musicplayer.MusicNotification
+
+class MusicPlayerService : Service(),
+    MediaPlayer.OnPreparedListener { // Service와 사용시 미디어 데이터를 가져오고 디코딩해야하는 경우 MediaPlayer 작업을 비동기 처리로 실행
+    private lateinit var notification: Notification
+    private var mediaPlayer: MediaPlayer? = null
+    private var position = 0
+    private val musicList = mutableListOf(R.raw.samples, R.raw.sample_2, R.raw.sample_3)
```

Comment:
```suggestion
    private val musicList = listOf(R.raw.samples, R.raw.sample_2, R.raw.sample_3)
```
musicList를 변경하지 않는다면 list 타입으로 변경해도 좋을 것 같네용